pkgs/build-support/expand-response-params/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation {
   '';
   installPhase = ''
     mkdir -p $prefix/bin
-    mv expand-response-params $prefix/bin/
+    mv expand-response-params${stdenv.hostPlatform.extensions.executable} $prefix/bin/
```

Comment:
Doesn't build phase need to use this with the `-o` too?

Reply:
> Doesn't build phase need to use this with the `-o` to `$CC` too?

Nope, It just makes a `.exe` automatically :shrug: